src/components/Foundation/Type/index.stories.js

```diff
@@ -185,3 +185,65 @@ storiesOf('foundation|Type', module)
       </div>,
     ),
   )
+  .add(
+    'Flourishes',
+    withInfo()(() =>
```

Comment:
I'm not a huge fan of the shit `withInfo` adds to the page.  Do you like it?

Reply:
Not really sure how to syntactically remove this - can you walk me through it?

Reply:
Remove `withInfo()(` from line 190
Remove `)` from line 249